source/evidencevariable/structuredefinition-EvidenceVariable.xml

```diff
@@ -568,5 +568,207 @@
 				<targetProfile value="http://hl7.org/fhir/StructureDefinition/Group"/>
 			</type>
 		</element>
+		<element id="EvidenceVariable.conditional">
+			<path value="EvidenceVariable.conditional"/>
+			<short value="Context in which the variable is observed"/>
+			<definition value="The context, situation, or parameters within which the variable is observed."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Expression"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.classifier">
+			<path value="EvidenceVariable.classifier"/>
+			<short value="Classification"/>
+			<definition value="Classification of the variable."/>
+			<min value="0"/>
+			<max value="*"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.datatype">
```

Comment:
The ticket indicates a dataStorage element should be added and that dataType and path should be sub-elements of dataStorage?

Reply:
You're correct. Fixed.